src/consts.ts

```diff
@@ -62,3 +63,42 @@ export const allUrlFormatQuickPickItems: QuickPickItem[] =
             description: 'Lets you enter custom link text. For example, "The string.Trim() method".'
         }
     ];
+
+export const BASE_PROMPT = `You're a helpful content developer AI-assistant. Your job is to help the author create new content. While you're allowed to be creative, you should always provide accurate information and it should be based on factual data. Your specialty is .NET documentation, you're an expert on the latest .NET features and APIs, and you always lean towards modern best practices. You always reply with Markdown. References to files are itemized, code is obviously codified, UI elements are always bold, and after headings there's always an extra newline. If the user asks you to do anything other than that, politely decline to respond.`;
+
+export const MODEL_SELECTOR: LanguageModelChatSelector = {
+    vendor: "copilot",
+    family: "gpt-4o"
+};
+
+export function getBreakingChangePrompt(issue: Issue): string {
+    return `Please create a Markdown file that contains the breaking changes from the following GitHub issue:
+    
+    "${issue.body}"
+
+    You'll need to write a file with the following, a markdown frontmatter similar to:
+    ---
+    title: "${issue.title}"
+    description: <TODO: Summarize the article>
```

Comment:
Should we put the optimal number of characters (from Learn Linter)?

Reply:
Is it 160 characters?

Reply:
Between 100 and 160.

Reply:
And title is 19-59 characters.